### PR TITLE
[6.3] Added API stubs for login page footer text

### PR DESCRIPTION
**Pull request by ntkathole**

### CODE REVIEW

File: tests/foreman/api/test_setting.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+:CaseImportance: High
+
+:Upstream: No
+"""
+from robottelo.decorators import tier1, stubbed
+from robottelo.test import APITestCase
+
+
+class SettingTestCase(APITestCase):
+    """Implements tests for Settings for CLI"""
```
Comment (san7ket) [inline tests/foreman/api/test_setting.py]: API?

File: tests/foreman/api/test_setting.py [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+:CaseComponent: API
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+"""
+from robottelo.decorators import tier1, stubbed
```
Comment (oshtaier) [inline tests/foreman/api/test_setting.py]: put alphabetically

File: tests/foreman/api/test_setting.py [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+from robottelo.test import APITestCase
+
+
+class SettingTestCase(APITestCase):
+    """Implements tests for Settings for API"""
+
+    @stubbed
+    @tier1
+    def test_positive_update_login_page_footer_text(self):
+
```
Comment (oshtaier) [inline tests/foreman/api/test_setting.py]: redundant line

File: tests/foreman/api/test_setting.py [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+        """Updates parameter "login_text" in settings
+
+        :id: 91c5373d-b928-419d-8509-761adf5b94b0
+
+        :steps:
+
+            1. Create login text with any string
```
Comment (oshtaier) [inline tests/foreman/api/test_setting.py]: not sure what is meant by that step, but from logic perspective - the whole `:steps:` section should be removed here

File: tests/foreman/api/test_setting.py [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+        """
+
+    @stubbed
+    @tier1
+    def test_positive_update_login_page_footer_text_without_value(self):
+        """Updates parameter "login_text" without any string (empty value)
+
+        :id: 7a56f194-8bde-4dbf-9993-62eb6ab10733
+
+        :steps:
```
Comment (oshtaier) [inline tests/foreman/api/test_setting.py]: same here

File: tests/foreman/api/test_setting.py [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+
+            1. Create login text without any string(empty value)
+
+        :expectedresults: Message on login screen should be removed
```
Comment (oshtaier) [inline tests/foreman/api/test_setting.py]: should we verify that from UI as these tests are API?

File: tests/foreman/api/test_setting.py [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+
+    @stubbed
+    @tier1
+    def test_negative_update_login_page_footer_text(self):
+        """Attempt to update parameter "Login_page_footer_text"
+            with invalid value(long length)
+
+        :id: fb8b0bf1-b475-435a-926b-861aa18d31f1
+
+        :steps:
```
Comment (oshtaier) [inline tests/foreman/api/test_setting.py]: same here

File: tests/foreman/api/test_setting.py (hunk 6 above)
Comment (ntkathole) [inline tests/foreman/api/test_setting.py]: @oshtaier  no need to verify that from UI...read that parameter from API and it should be empty.

**Comment by ntkathole** [comment]
@oshtaier ACK pending

File: tests/foreman/api/test_setting.py [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+        """
+
+    @stubbed
+    @tier1
+    def test_positive_update_login_page_footer_text_without_value(self):
+        """Updates parameter "login_text" without any string (empty value)
+
+        :id: 7a56f194-8bde-4dbf-9993-62eb6ab10733
+
+        :expectedresults: Message on login screen should be removed
```
Comment (oshtaier) [inline tests/foreman/api/test_setting.py]: if we `no need to verify that from UI...read that parameter from API and it should be empty.`, so please change information here... to `read that parameter from API and it should be empty.` or better `login_text has empty value after update'

**Comment by ntkathole** [comment]
@oshtaier updated changes..thanks

**Pull request merged**